### ISSUE TITLE
Clicking start on menu crashes (space bar allows you to play the game still)

### ISSUE THREAD

**Issue by jbtule** [dummey/CS50-Cohort-Project-Redux#26]
```
menu_scene.rb:46:in `update': undefined method `intersect?' for nil:NilClass (NoMethodError)
    from Asteroids.rb:25:in `update'
    from Asteroids.rb:53:in `<main>'
```

**Comment by jbtule** [comment]
I'm assigning this to you @dummey , but reassign as you see fit.

**Comment by dummey** [comment]
Oh yups, that is me

**Comment by dummey** [comment]
Fixed in 665596dcb4447e2de089d7ce8c59f37029b02fe8
